lms/djangoapps/course_home_api/progress/tests/test_views.py

```diff
@@ -127,7 +127,7 @@ def test_end(self):
     @override_waffle_flag(COURSE_HOME_MICROFRONTEND_PROGRESS_TAB, active=True)
     def test_user_has_passing_grade(self):
         CourseEnrollment.enroll(self.user, self.course.id)
-        self.course._grading_policy['GRADE_CUTOFFS']['Pass'] = 0
+        self.course.grade_cutoffs = {'Pass': 0}
```

Comment:
This feels like a bigger change? Just a miss originally about how to access grade_cutoffs?

Reply:
Yeah I think just a miss. Linter was complaining about using a private accessor and I looked at the class - the above two ways are equivalent (except the proper public API way also updates another in-sync field)